src/js/views/maps/ToolbarView.js

```diff
@@ -203,16 +202,20 @@ define([
           label: "Share",
           icon: "link",
           action(view) {
-            if (view.$el.find(`.${view.classes.shareUrlView}`).length) return;
-
             const title = this.linkEl.querySelector(
               `.${view.classes.linkTitle}`,
             );
+
+            if (title.classList.contains(view.classes.linkTitleHidden)) {
```

Comment:
does this suggest a scenario when a ShareUrlView already exists? would it be more straightforward to check for that directly?

Reply:
I didn't want to use the class name of ShareUrlView as it's an implementation detail of the view, but it's clearer so I will check for its presence rather than checking for this hidden link class name.